engine/src/main/scala/cromwell/engine/db/ExecutionInfosByExecution.scala

```diff
@@ -0,0 +1,101 @@
+package cromwell.engine.db
+
+import cromwell.engine._
+import cromwell.engine.backend._
+import cromwell.engine.db.slick.{ExecutionInfo, Execution}
+import wdl4s.values.WdlFile
+
+case class ExecutionInfosByExecution(execution: Execution, executionInfos: Seq[ExecutionInfo]) {
+
+  lazy val callLogs: Option[CallLogs] = {
+    import ExecutionInfosByExecution._
+
+    case class Accumulator(stdout: Option[WdlFile], stderr: Option[WdlFile], logs: Map[String, WdlFile])
+
+    val acc = executionInfos.foldLeft(Accumulator(None, None, Map.empty)) { (acc, executionInfo) =>
+      executionInfo.key.split("_", 2) match {
+        case Array(CallLogPrefix, StdoutSuffix) => acc.copy(stdout = executionInfo.value map {
```

Comment:
nm, this doesn't seem to play as nicely with the defaulted arg on `WdlFile` as I had hoped :frowning: